source/_components/zha.markdown

```diff
@@ -105,6 +117,15 @@ To add new devices to the network, call the `permit` service on the `zha` domain
 
 In case you want to add Philips Hue bulbs that have previously been added to another bridge, have a look at: [https://github.com/vanviegen/hue-thief/](https://github.com/vanviegen/hue-thief/)
 
+## Adding devices
+Go to the `Configuration` page and select the `ZHA` element that was added by the configuration steps above
+
+Click `ADD DEVICES` to start a scan for devices.
+
+Reset your zigbee devices according to your manufacturer instructions (lights turn on and off upto 10 times, switches using the reset pin)
+
+Devices will now become visible and can be added to `Areas`
```

Comment:
```suggestion
```

Removing this sentence does not change the text in any way, so let us remove it.